releases/8.0.0.md

```diff
@@ -223,6 +224,8 @@ JavaScript:
     python manage.py updateproject
     ```
 
+    Verify in `<project_name>/apps.py` that you no longer have a ``ready()`` method.
```

Comment:
seems like this should have it's own number? also double-ticked?

Reply:
for whatever reason the vs code preview treats single & double the same way, but I think you're right 👍 

Sounds good on a new numbered bullet.